src/pages/main/components/itemList/ItemList.js

```diff
@@ -1,7 +1,131 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Item from './Item';
+import CategoryComponent from './Category';
+import CATEGORY_LIST from './categoriesListData';
+import BANNER_LIST from './BannerListData';
+import BannerComponent from './Banner';
 
-function ItemList() {
-  return <div>itemList</div>;
-}
+const ItemList = () => {
+  const [productsList, setProductsList] = useState([]);
 
+  useEffect(() => {
+    fetch('http://localhost:3000/data/productsData.json', {
```

Comment:
`http://localhost:3000`은 생략 가능합니다. 포트 번호가 바뀔 때마다 3000을 3001, 3002와 같이 일일이 바꿔야 하는 번거로움이 있기 때문에 생략하는 것이 좋습니다!
```suggestion
    fetch('/data/productsData.json', {
```

Reply:
수정했습니다!